### PR TITLE
Modify scripts for weekly_tests

### LINKED ISSUE #956: Update regression test scripts for weekly tests
## Description
The current regression test scripts are designed to either create a new baseline or compare output files with those from previous baseline automatically after the models are completed.

For the weekly tests, we only want to know if the models are completed successfully. To do so, we need to update several scripts.

## Solution
The following scripts will be updated:
rt.sh
run_test.sh
rt_utils.sh

### PR DESCRIPTION
**Pull request by binli2337**
# PR Checklist

- [ ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ ] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description
The current regression test scripts are designed to either create a new baseline or compare output files with those from previous baseline automatically after the models are completed.

For weekly tests, we only want to know whether the models can be run successfully. The following scripts are updated for the weekly tests:
rt.sh, run_test.sh and rt_utils.sh

### Issue(s) addressed
- fixes #956 

## Testing
How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [ ] hera.intel
- [ ] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [ ] CI

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs

### CODE REVIEW

**Comment by DusanJovic-NOAA** [comment]
If I understand correctly the idea here is to determine (based on the configuration file name) whether or not to run `check_results` function. Wouldn't it be simpler to introduce a command line switch which can then be used to either skip `check_results` or not, and avoid adding 6 or 7 `if [[ ${WEEKLY_TEST}} ]]` tests scattered around three files. Using a switch has an advantage that it can be used with any of `.conf` files, not just those having string `weekly` in it.

**Comment by MinsukJi-NOAA** [comment]
> If I understand correctly the idea here is to determine (based on the configuration file name) whether or not to run `check_results` function. Wouldn't it be simpler to introduce a command line switch which can then be used to either skip `check_results` or not, and avoid adding 6 or 7 `if [[ ${WEEKLY_TEST}} ]]` tests scatter around three files. Using a switch has an advantage that it can be used with any of `.conf` files, not just those having string `weekly` in it.

Do we want to have a separate `.conf` file for the weekly test OR do we want an additional column (for weekly test) in the current `rt.conf` file? I think the latter may be better in terms of maintenance.

**Comment by binli2337** [comment]
Right now, there is only one weekly test. From my understanding, we will add more tests in the future, such as the 8 35-day tests.  It might be better to have a separate configuration file, such as rt_weekly.conf.

We also need a separate log file since the log file for regular tests might be updated a few times a week while the log file for weekly tests is only updated once a week.

**Review by DusanJovic-NOAA — APPROVED** [review]

File: tests/run_test.sh [hunk 1]
```diff
@@ -237,7 +237,9 @@ else
 
 fi
 
-check_results
+if [[ $skip_check_results = false ]]; then
```
Comment (climbfuji) [inline tests/run_test.sh:240]: Double negating is always confusing.

**Comment by binli2337** [comment]
Merged in PR #954

**Pull request closed without merging**